ee/debug/checkups/init_logs_windows.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func writeInitLogs(ctx context.Context, logZip *zip.Writer) error {
-	cmdStr := `Get-WinEvent -FilterHashtable @{LogName='Application'; ProviderName='launcher'} | ConvertTo-Json`
+	cmdStr := `Get-WinEvent -FilterHashtable @{LogName='Application'; ProviderName='launcher'} | ForEach-Object { $_.Message }`
```

Comment:
this works pretty well except for powershell adding a new line between each log, we could remove with some post processing of the output but ... 🤷

Reply:
If you want to monkey here, you could also add `Get-EventLog  -Newest 10 -LogName System -Source "Service Control Manager" -Message "*launcherkolidek2svc*" | select -ExpandProperty message`